src/init.cpp

```diff
@@ -760,27 +774,32 @@ bool AppInit2(Config &config, boost::thread_group &threadGroup, CScheduler &sche
     // cost to you of processing a transaction.
     if (mapArgs.count("-minlimitertxfee"))
     {
-        CAmount n = 0;
-        double minrelaytxfee;
+        CAmount nAmt = 0;
+        double minrelaytxfee = 0;
+
         try
         {
-            minrelaytxfee = boost::lexical_cast<double>(mapArgs["-minlimitertxfee"]) / 100000;
+            dMinLimiterTxFee.value = boost::lexical_cast<double>(mapArgs["-minlimitertxfee"]);
         }
         catch (boost::bad_lexical_cast &)
         {
             return InitError(
-                _("ERROR: an incorrect value was specified for -minlimitertxfee.  Please check value and restart."));
+                strprintf(_("Invalid amount for -minlimitertxfee=<amount>: '%s'"), mapArgs["-minlimitertxfee"]));
         }
 
+        // calculate the starting minrelaytxfee. We get this from the minlimitertxfee tweak
+        minrelaytxfee = dMinLimiterTxFee.value / 100000;
+
         ostringstream ss;
         ss << fixed << setprecision(8);
         ss << minrelaytxfee;
-        if (ParseMoney(ss.str(), n))
-            ::minRelayTxFee = CFeeRate(n);
+        if (ParseMoney(ss.str(), nAmt))
+            ::minRelayTxFee = CFeeRate(nAmt);
```

Comment:
This code sets minRelayTxFee only if minlimitertxfee is explicitly set by the user.  Instead, can we set minRelayTxFee based the value minlimitertxfee no matter what --presumably the default minlimitertxfee results in the expected default minRelayTxFee.  Otherwise we get into a situation where if minlimitertxfee is set to the default value, the behavior is different than if minlimitertxfee not touched.

Reply:
all taken out, not needed anymore